src/views/Lottery/components/Hero.tsx

```diff
@@ -1,25 +1,41 @@
 import React from 'react'
 import styled from 'styled-components'
-import { Heading, Text } from '@pancakeswap-libs/uikit'
+import { Heading, Text, ArrowForwardIcon, useMatchBreakpoints } from '@pancakeswap-libs/uikit'
 import useI18n from 'hooks/useI18n'
 import Container from 'components/layout/Container'
-import LotteryProgress from './LotteryProgress'
 
 const Title = styled(Heading).attrs({ as: 'h1', size: 'xl' })`
-  color: ${({ theme }) => theme.colors.secondary};
+  color: #ffffff;
   margin-bottom: 24px;
+  text-shadow: 2px 2px 2px #00000040;
 `
 
-const Blurb = styled(Text)`
-  color: #ffffff;
-  font-size: 20px;
+const ComeBack = styled(Text)`
+  background: -webkit-linear-gradient(#ffd800, #eb8c00);
```

Comment:
Why the prefix?

Reply:
Styled components automatically adds the vendor prefixes.

Reply:
I'm not sure why but it doesn't work without it. With prefix works on Chorm, Firefox and Safari.

Reply:
That what it shows if I remove the prefixes. I can take a look at it later if you don't mind, gonna remove the nextdraw page in later PR. Might need some special wrapper for it to work without prefix or some stupid css thing like that....
![Screenshot 2021-04-30 at 12 00 29](https://user-images.githubusercontent.com/81824236/116673298-e4ea6380-a9ab-11eb-9cdc-c899636d5749.png)